server/handlers/handlers.go

```diff
@@ -194,21 +211,21 @@ func (ah *ActivityHandler) handleCreatedActivity(activityIds msteams.ActivityIds
 			}
 		}
 
-		return
+		return discardedReasonOther
```

Comment:
@manojmalik20 do you mind to take a look to this. It looks like a bug. From my understanding of the code here is, if the user is a guest and the guest user are not synced, we stop processing the event here. My expectation is we still process the event.

Reply:
@jespino This was the requirement from ServiceNow. If a guest user is posting a message on Teams, and the setting `SyncGuestUsers` is off, then we have to discard all his activities. Although, I think that this may be a bug in the "UpdatedActivity" function as we're not doing this there. I'll check this out.